package.json

```diff
@@ -69,7 +69,7 @@
     "nyc": "15.1.0",
     "prettier": "2.7.1",
     "redis": "4.0.6",
-    "sinon": "13.0.2",
+    "sinon": "14.0.0",
```

Comment:
Sinon 14 drops support for Node 12, which is something we support in our build matrix in this project. Node 12 has hit EOL for security updates though so I'm tempted to drop it and consider merging this unless there's a strong reason to support EOL node versions here

@ivarconr WDYT?